harness/adapters/google.py

```
@@ -0,0 +1,206 @@
+            base_delay = float(os.getenv("GOOGLE_BACKOFF_BASE", 1.0))
+            attempt = 0
+            last_exc: Exception | None = None
+            
+            while attempt < max_attempts:
+                attempt += 1
+                # Parameter adaptation loop
+                for _ in range(3):
+                    try:
+                        resp = self.client.models.generate_content(**params)
+                        break
+                    except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)